import_export/widgets.py

```diff
@@ -8,7 +9,14 @@
 from django.utils import timezone
 from django.utils.dateparse import parse_duration
 from django.utils.encoding import force_str, smart_str
+from django.utils.translation import get_language, to_locale
 
+def format_number(value):
```

Comment:
Please can you add a new line above this line to fix the test failure (failing due to isort).

Alternatively you can run `isort .` to correct

Reply:
Done.